Dockerfile

```
@@ -27,7 +27,7 @@ RUN pip install --no-cache-dir pip install "poetry ~= 1.8.2" \
     && poetry build --format wheel --no-ansi
 
 # Stage 3: Build the application.
-FROM node:18 AS build
+FROM node:19 AS build
```

```suggestion
FROM node:20 AS build
```

what about nodejs 20 ? its current LTS version. Does it solve the problem ?

---

Unfortunately after recent bios update (another issue with my laptop) sth broke on my linux and graphic interface doesn't work. I'm moving back to windows.

---

Then make some check on WSL, it should work, also when you start container from Windows